lib/core/src/Cardano/Wallet/Primitive/AddressDiscovery.hs

```diff
@@ -52,12 +54,12 @@ import Cardano.Wallet.Primitive.Types
 -- In practice, we will need a wallet that can support both, even if not at the
 -- same time, and this little abstraction can buy us this without introducing
 -- too much overhead.
-class IsOurs s where
+class IsOurs s entity where
```

Comment:
I would still call this parameter `address`. `entity` is more general than it needs to be.

Reply:
> I would still call this parameter `address`. `entity` is more general than it needs to be.

Ah, that naming suggestion came from a conversation with @KtorZ. The plan would be to eventually move this class to another module, to reflect the new level of generality. We would delay moving this class until a recovery week.